nilearn/plotting/surf_plotting.py

```diff
@@ -20,7 +20,7 @@
 from nilearn.surface.surface import _check_mesh
 from nilearn._utils import check_niimg_3d, fill_doc
 from nilearn.plotting.js_plotting_utils import colorscale
-from nilearn.plotting.html_surface import _get_vertexcolor
+from nilearn.plotting.html_surface import _get_vertexcolor, _mix_colormaps
```

Comment:
In another PR, it would be nice to move these functions out of `html_surface` and rename them if they are to be used by other modules. eg `cm.mix_colormaps` or `plotting._utils.mix_colormaps` instead of `plotting.html_surface._mix_colormaps`